fastlane/lib/fastlane/lane_manager.rb

```diff
@@ -103,30 +103,51 @@ def self.print_table(actions)
     # Lane chooser if user didn't provide a lane
     # @param platform: is probably nil, but user might have called `fastlane android`, and only wants to list those actions
     def self.choose_lane(ff, platform)
-      loop do
-        UI.error "You must provide a lane to drive. Available lanes:"
-        available = ff.runner.available_lanes(platform)
+      available = ff.runner.lanes[platform].to_a
+      if !available.empty?
```

Comment:
Instead you could use 

```ruby
if available.empty?
  UI.error "It looks like you don't have any lanes to run just yet. Check out how to get started here: https://github.com/fastlane/fastlane 🚀"
  exit
end
loop do
...
```

this way we don't have to intend all the code, it's cleaner :+1: